test/circle/lint.sh

```diff
@@ -53,15 +53,29 @@ yamllinter() {
   fi
 }
 
+# Validate the Chart.yaml
+validate_chart_yaml() {
+
+  run yamale -s test/circle/yaml-schemas/Chart.yaml ${1}/Chart.yaml
+
+  echo "Validating maintainers names"
+
+  for name in $(yq -r '.maintainers|.[]|.name' ${1}/Chart.yaml); do
+    if [ $(curl -s -o /dev/null -w "%{http_code}\n" -If https://github.com/${name}) -ne 200 ]; then
+      echo "Error: ${name} is not valid GitHub account"
+      exitCode=1
+    fi
+  done
+}
+
 # include the semvercompare function
 curDir="$(dirname "$0")"
 source "$curDir/../semvercompare.sh"
-
-git remote add k8s https://github.com/kubernetes/charts
+#git remote add k8s https://github.com/kubernetes/charts
```

Comment:
Was this removed intentionally?

Reply:
I modified the file for local testing. this is post revert leftovers